packages/concerto-core/lib/introspect/modelfile.js

```diff
@@ -41,11 +39,12 @@ class ModelFile {
      * Use the ModelManager to manage ModelFiles.
      * @param {ModelManager} modelManager - the ModelManager that manages this
      * ModelFile
-     * @param {string} definitions - The DSL model as a string.
+     * @param {object} ast - The DSL model as a string.
+     * @param {string} [definitions] - The (optional) CTO model as a string.
```

Comment:
I think I lean towards removing [definitions] altogether. It will use memory to store it and it doesn't seem to add value can could be out of sync with the AST.

Reply:
Agreed. There are parts of the model manager API still relying on this though so it's a bigger change. I'll fold this into the next PR.